cglims/api.py

```diff
@@ -11,6 +11,49 @@ def connect(config):
     api = ClinicalLims(config['host'], config['username'], config['password'])
     return api
 
+class ClinicalSample:
+
+    def __init__(self, sample):
+        """ Wrapper around the genologics Sample class
+
+        Args:
+            sample (genologics.Sample): the sample instance to extend
+        """
+        self.lims = sample
+        self._pipeline = None
+        self._apptag = None
```

Comment:
I'm thinking that we'll have properties/methods for the heavier calculations anyways

The "__init__" part of the ApplicationTag class shouldn't grow in complexity

At that point it might be okey to instatiate it right away